optuna/pruners/_hyperband.py

```diff
@@ -56,6 +56,11 @@ class HyperbandPruner(BasePruner):
         6 in most use cases).　Please see Section 3.6 of the `original paper
         <http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`_ for the detail.
 
+    .. note::
+        ``HyperbandPruner`` assumes that ``step`` starts at zero when it calculates the resource
+        for each trial. Please ensure that ``step`` is zero-based numbering when you report the
```

Comment:
A grammatical detail. I believe we can skip "numbering" altogether.
```suggestion
        for each trial. Please ensure that ``step`` is zero-based when you report the
```